auth.go

```diff
@@ -472,14 +472,11 @@ func (x *Central) Login(username, password string) (sessionkey string, token *To
 		Identity: user.getIdentity(),
 		UserId:   user.UserId,
 	}
+
 	sessionExpiry := time.Now().Add(x.NewSessionExpiresAfter)
-	if x.PasswordExpiresAfter != 0 {
-		userPasswordExpiry := user.PasswordModifiedDate.Add(x.PasswordExpiresAfter)
-		if userPasswordExpiry.Before(sessionExpiry) {
-			token.Expires = userPasswordExpiry
-		} else {
-			token.Expires = sessionExpiry
-		}
+	userPasswordExpiry := user.PasswordModifiedDate.Add(x.PasswordExpiresAfter)
```

Comment:
Why did you remove the check for `x.PasswordExpiresAfter != 0`. That seems like a valid thing to keep.